techmatters/serverless
# CHI-1260: Infer staging accounts to deploy

## Description

* Create a local action to read hewlpline short codes based on entries in the SSM parameter store
* Automatically infer which staging accounts to deploy to based on the existence of the SSM parameters created as part of the account setup, removing the need for a separately maintained account list in the repo

### Checklist
- N/A New tests added

### Related Issues
CHI-1260

### Verification steps

Run the 'Deploy All Staging Accounts' workflow from this branch & ensure the expected accounts are successfully deployed
